README.md

```diff
@@ -102,7 +102,29 @@ Validators update the miner list hourly. It's normal not to observe requests in
 nc -vz [Your Miner IP] [Port]
 ```
 
-# ... to be updated ...
+## Validation
+
+### Setup
+To install all dependencies and generate the run configuration execute the following command in the project base directory:
+```
+make validator
+```
+
+### Startup
+To start the validator service execute the following command in the project base directory:
+```
+make start_validator
```

Comment:
Running this leads to `ModuleNotFoundError: No module named 'neurons'`

Reply:
Should also solved